### ISSUE TITLE
Demo

### ISSUE THREAD

**Issue by bruno12mota** [relax/relax#26]

**Comment by magalhas** [comment]
The demo env is not in fact fully achieved.

**Comment by bruno12mota** [comment]
Yup, should save changes locally for each user instead of saving to server

**Comment by MoOx** [comment]
I just got "Unauthorized" when using demo/demo here http://demo.getrelax.io/admin/ (the "login" button redirect me to http://demo.getrelax.io/admin/login).

**Comment by bruno12mota** [comment]
Someone removed the demo user, sorry about that, created it again

**Comment by MoOx** [comment]
Nice. You should add a trick to prevent this in the future :)

**Comment by MoOx** [comment]
Or maybe you should reset demo db every 30min and add a banner on the demo. Something like that. Because it's weird to get this page empty http://demo.getrelax.io/

**Comment by magalhas** [comment]
We would like to have the demo writing and reading from local storage but it's getting hard to find time for it.

**Comment by MoOx** [comment]
It's seems that a smart guy already deleted (again) demo user...

**Comment by bruno12mota** [comment]
I'm adding a flag in the rc file now to prevent the removal of users for now

**Comment by augbog** [comment]
:+1: Thanks @bruno12mota I also noticed the demo user got deleted again.

**Comment by bruno12mota** [comment]
Just fixed it, we updated the demo by the current version, had to make a little database update. Thanks for warning :+1:

**Comment by bruno12mota** [comment]
So should we provide a solution that doesn't allow saves and mutations? Or should we make a timed reset to the database?

**Comment by DanielRuf** [comment]
I'm for a timed reset.

**Comment by augbog** [comment]
I am also for a timed reset but is there a way we can just immediately reset when the demo user gets deleted? That way there won't be awkward periods of time when the demo user is deleted and we have to wait for the timed reset.
